scripts/write_deployinst_input.py

```diff
@@ -201,43 +200,86 @@ def reactor_render(list, template, mox_template, output_file):
         jinja template for mox reactor file
     output_file: str
         name of output file
+    is_cyborg: bool
+        if True, uses Cyborg templates
 
     Returns
     -------
     The reactor section of cyclus input file
 
     """
-
+    if is_cyborg:
```

Comment:
There is a much cleaner way to do this. The behavior in this if statement is identical to the behavior to the else statement except for the addition of _cyborg to each filename... can you think of a cleaner way to conduct this task? Similarly, the difference between the template tasks is explicitly the name of the reactor! This should be a dictionary, not a list of identities.